chord_metadata_service/mohpackets/data/README.md

```diff
@@ -1,7 +1,49 @@
 # About
 
-This folder contains synthetic data for MoH models.
+This folder contains synthetic data for **MoH models**.
 
-The **schemas** contains the blueprints to generate the data using [Mockaroo](https://www.mockaroo.com/) ("RESTORE FROM BACKUP...")
+The **schemas** contains the blueprints to generate the data.
 
-The **fixtures** contains the synthetic data in JSON format for quick import into PostgreSQL.
+The **synthetic_data** folder contains the generated data.
```

Comment:
There is currently data in the synthetic_data folder in the repo: is that an already-created instance of this test data?

Reply:
yes, I included the ready-made data in the folder to make it easy for everyone, so we can just ingest it.